src/components/Overlay.tsx

```diff
@@ -118,6 +117,7 @@ export default class JoyrideOverlay extends React.Component<OverlayProps, State>
       top,
       transition: 'opacity 0.2s',
       width: Math.round((elementRect?.width ?? 0) + spotlightPadding * 2),
+      ...(isLegacy() ? styles.spotlightLegacy : styles.spotlight),
```

Comment:
This will allow us to override styles like `top` and `left` properties of spotlight

Reply:
Hey @manishPh 

Things aren't that simple... Validation failed.

Remember to run it locally before committing.  The `npm run validate` should run in the pre-commit hook.
Did you disable it?

Reply:
@gilbarbara I made sure the `validate` is working locally. Happy to discuss implementation approach if you have suggestion.